src/utils/get-device-size.js

```diff
@@ -0,0 +1,23 @@
+import { readSync } from 'read-file-relative';
+import find from 'array-find';
+
+
+const DEVICE_DATABASE = JSON
+    .parse(readSync('../../data/devices.json'))
+    .map(device => {
+        return {
+            name:           device['Device Name'].toLowerCase().split(' ').join(''),
+            portraitWidth:  Number(device['Portrait Width']),
+            landscapeWidth: Number(device['Landscape Width'])
+        };
+    })
+    .filter(device => !isNaN(device.portraitWidth) && !isNaN(device.landscapeWidth));
+
+
+export default function (deviceName) {
+    deviceName = deviceName.split(' ').join('');
+
+    var { portraitWidth, landscapeWidth } = find(DEVICE_DATABASE, device => device.name.indexOf(deviceName) !== -1);
+
+    return { portraitWidth, landscapeWidth };
```

Comment:
Why we need intermediate `{ portraitWidth, landscapeWidth }` vars. Just return `find..`